common/osOpen_windows.go

```diff
@@ -23,6 +25,11 @@ func OSOpenFile(name string, flag int, perm os.FileMode) (*os.File, error) {
 	return file, nil
 }
 
+// TODO: os.Stat(name string) already uses backup semantics.  Can/should we just use that?
+//  That would work around the issue where here we are using OSOpen, which technically returns a bad result for
+//  directories (but which we seem to get away with in the os.Stat(handle) call - which goes down the "I am a file" code
+//  path, but still returns a valid result for a directory.
+//  See also todo on OsOpen, and edit or remove it
 func OSStat(name string) (os.FileInfo, error) {
```

Comment:
It would make sense to remove this, if os.Stat() works just fine here.

Reply:
Covered in a separate PR